packages/configs/tsconfig.node.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "./tsconfig.base.json",
+    "compilerOptions": {
+        "lib": ["es2021"],
+        "module": "commonjs",
+        "target": "es2021"
+    },
+
```

Comment:
If in `packages/configs/eslint-common-backend.json` we are referring to specific paths, maybe here we can also keep `"exclude": ["**/__mocks__/*", "test"]` (which is present in both - Stage and Composer - backend TS configs)?

Reply:
I tried this but it didn't work - looks like the `exclude` property is not picked up from the extended configuration.